src/components/PlatformNavigator/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState, useRef, useEffect } from 'react';
+import { Button, Flex, Text, View } from '@aws-amplify/ui-react';
+import { IconChevron } from '@/components/Icons';
+import { frameworks } from '@/constants/frameworks';
+import { InfoPopover } from './InfoPopover';
+import Link from 'next/link';
+import classNames from 'classnames';
+import { useClickOutside } from '@/utils/useClickOutside';
+
+export function PlatformNavigator({ currentPlatform }) {
```

Comment:
@jacoblogan , curious if we need to close the drop down once we select the value, please let me know.
![image](https://github.com/aws-amplify/docs/assets/124720934/1c9d631b-b0ef-49a4-8ddf-2951a6359ab9)